examples/nlp/gpt/train_reward_model.py

```diff
@@ -131,6 +131,8 @@ def main(cfg) -> None:
 
     logger.log_hyperparams(OmegaConf.to_container(cfg))
 
+    timer = Timer(cfg.exp_manager.get("max_time_per_run", None))
```

Comment:
nit but just a reminder that you can simply write
```suggestion
    timer = Timer(cfg.exp_manager.get("max_time_per_run"))
```